cypress/integration/accessibility.spec.js

```diff
@@ -8,7 +8,26 @@ import { formatDate } from '../../src/utils/date';
 describe('Accessibility Requests', () => {
   it('can create a request and see its details', () => {
     cy.localLogin({ name: 'A11Y' });
-    cy.accessibility.create508Request();
```

Comment:
This is the test where we'll actually go all the way through the creation process in the UI to make sure everything works.